axelrod/tests/unit/test_strategy_transformers.py

```diff
@@ -349,6 +420,7 @@ def test_mixed(self):
         probability = (0, 0, 1)  # Note can also pass tuple
         strategies = [axelrod.TitForTat, axelrod.Grudger, axelrod.Defector]
         MD = MixedTransformer(probability, strategies)(axelrod.Cooperator)
+        self.assertFalse(MD.classifier["stochastic"])
 
         p1 = MD()
         # Against a cooperator we see that we only cooperate
```

Comment:
i've been reading through this to try and wrap my head around decorators, which drive me up the wall.  it has helped but it's also hard reading.  This comment that sneaked in from a copy-paste made me laugh.